gradle/plugins/src/main/kotlin/com.hedera.gradle.jpms-modules.gradle.kts

```diff
@@ -67,7 +67,8 @@ jvmDependencyConflicts.patch {
     module("com.google.guava:guava") {
         (annotationLibraries -
                 "com.google.code.findbugs:jsr305" -
-                "com.google.errorprone:error_prone_annotations")
+                "com.google.errorprone:error_prone_annotations" -
+                "org.checkerframework:checker-qual")
```

Comment:
Turns out we need the annotations from `checker-qual` on the classpath for Javadoc. Therefore, we no longer remove them from the transitive dependencies of Guava.